spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -97,7 +97,7 @@ public TaskExecutionController(TaskExplorer explorer, TaskService taskService,
 	@ResponseStatus(HttpStatus.OK)
 	public PagedResources<TaskExecutionResource> list(Pageable pageable,
 			PagedResourcesAssembler<TaskJobExecutionRel> assembler) {
-		Page<TaskExecution> taskExecutions = explorer.findAll(pageable);
+		Page<TaskExecution> taskExecutions = this.taskService.findAll(pageable);
```

Comment:
@cppwfs I view service == business logic.  Controller == back end display logic.  To be *really* honest, you could argue that this is a security concern and should be handled by a `ServletFilter` (applying this logic across all requests)...